src/lib.rs

```diff
@@ -1610,6 +1615,20 @@ impl RtcConfig {
         self
     }
 
+    /// Set DTLS certification.
+    pub fn set_dtls_certification(mut self, dtls_cert: DtlsCert) -> Self {
+        self.dtls_cert = dtls_cert;
+        self
+    }
```

Comment:
Move this to after `dtls_cert` and rename it to `set_dtls_cert`.

Reply:
Done